packages/plugins/plugin-sheet/src/translations.ts

```diff
@@ -16,7 +16,12 @@ export default [
         'toolbar left label': 'Align left',
         'toolbar left center': 'Align center',
         'toolbar left right': 'Align right',
+        'selection overlaps existing comment label': 'Selected cell already has a comment',
+        'comment label': 'Add comment',
+        'comment ranges not supported label': 'Commenting on ranges is not supported (yet)',
```

Comment:
`not yet supported.`
End messages (not menus, etc.) with periods in user messages?